server/src/useCases/report/reportSave/handleSaveRequest.ts

```diff
@@ -46,23 +52,39 @@ const RequestSchema = {
 };
 
 export default class ReportUploadRequestHandler {
-  public req: any;
-  public res: any;
-  public dbConnect: any;
+  public req: CommonTypes.MulterRequest;
+  public res: Response;
+  public dbConnect: Model<Document>;
 
   constructor(saveRequest: any, response: any) {
     this.req = saveRequest;
     this.res = response;
     this.dbConnect = saveRequest.app.locals.reportCollection;
   }
 
-    async saveData(repository: any): Promise<boolean> {
+  async saveData(repository: ReportInterfaces.ReportRepository): Promise<boolean> {
     if (!this.isRequestValid()) {
       return false;
     }
-    const dataToSave = new TestCaseBuilder(await this.loadData()).buildExecutionResult();
-    const productMetaData = await this.loadProductInfo();
-    await this.jsonSave(repository, dataToSave, productMetaData);
+    const executionResult: any = new TestCaseBuilder(await this.loadData()).buildExecutionResult();
```

Comment:
please move await this..[..] to new line

Reply:
done